### ISSUE TITLE
Generating using @ngrx/schematics throws "Expected a synchronous delegate but got an asynchronous one." error

### ISSUE THREAD

**Issue by IslombekMe** [nrwl/nx#15120]
### Current Behavior

I am in the middle of creating Ngrx files for my projects/libraries and, when running the generate command, I get the following error:
```
$ /Users/user/Downloads/project1/node_modules/.bin/nx generate @ngrx/schematics:store assets --module=domain --project=fleet-accounting-domain --minimal --no-interactive --dry-run

>  NX  Generating @ngrx/schematics:store

SynchronousDelegateExpectedException [Error]: Expected a synchronous delegate but got an asynchronous one.
    at SyncDelegateHost._doSyncCall (/Users/user/Downloads/project1/node_modules/@angular-devkit/core/src/virtual-fs/host/sync.js:39:19)
    at SyncDelegateHost.read (/Users/user/Downloads/project1/node_modules/@angular-devkit/core/src/virtual-fs/host/sync.js:57:21)
    at LazyFileEntry._load (/Users/user/Downloads/project1/node_modules/@angular-devkit/schematics/src/tree/host-tree.js:252:80)
    at LazyFileEntry.get content [as content] (/Users/user/Downloads/project1/node_modules/@angular-devkit/schematics/src/tree/entry.js:34:55)
    at HostTree.read (/Users/user/Downloads/project1/node_modules/@angular-devkit/schematics/src/tree/host-tree.js:211:30)
    at getWorkspace (/Users/user/Downloads/project1/node_modules/@ngrx/schematics/schematics-core/utility/config.js:13:29)
    at getProject (/Users/user/Downloads/project1/node_modules/@ngrx/schematics/schematics-core/utility/project.js:6:47)
    at getProjectPath (/Users/user/Downloads/project1/node_modules/@ngrx/schematics/schematics-core/utility/project.js:19:19)
    at /Users/user/Downloads/project1/node_modules/@ngrx/schematics/src/store/index.js:133:61
    at callRuleAsync (/Users/user/Downloads/project1/node_modules/@angular-devkit/schematics/src/rules/call.js:78:24)
```

### Expected Behavior

It should successfully execute the generator

### Steps to Reproduce

1. Open any nx project.
2. Run generate with any of Ngrx Schematics, e.g. `nx generate @ngrx/schematics:store`

### Nx Report

```shell
Node : 16.16.0
   OS   : darwin arm64
   yarn : 1.22.19
   
   nx                      : 15.7.2
   @nrwl/jest              : 15.7.2
   @nrwl/linter            : 15.7.2
   @nrwl/workspace         : 15.7.2
   @nrwl/angular           : 15.7.2
   @nrwl/cli               : 15.7.2
   @nrwl/cypress           : 15.7.2
   @nrwl/devkit            : 15.7.2
   @nrwl/eslint-plugin-nx  : 15.7.2
   @nrwl/js                : 15.7.2
   @nrwl/storybook         : 15.7.2
   @nrwl/tao               : 15.7.2
   @nrwl/webpack           : 15.7.2
   typescript              : 4.8.4
   ---------------------------------------
   Community plugins:
   @ngneat/input-mask    : 5.4.0
   @ngrx/component-store : 15.3.0
   @ngrx/data            : 15.3.0
   @ngrx/effects         : 15.3.0
   @ngrx/entity          : 15.3.0
   @ngrx/router-store    : 15.3.0
   @ngrx/store           : 15.3.0
   @ngrx/schematics      : 15.3.0
   @ngrx/store-devtools  : 15.3.0
   @storybook/angular    : 6.5.15
```

**Comment by AgentEnder** [comment]
This may have been fixed by #15137, can you confirm @leosvelperez ?

**Comment by leosvelperez** [comment]
That's correct. This was fixed in #15137 and the fix will be released soon. Keep an eye on the releases to get it.
